Remove FIPS note that referred to an old release

btm opened the pull request:
ChefDK 1.3.40 had a FIPS bug on Windows which was fixed in omnibus-software#823
and released ten days later in ChefDK 1.3.43. This note is cruft and should
have been removed then.

cwebberOps commented:
@jamescott lets work together to merge this.

@metadave if you want to submit this change via delivery, it is easier to merge on our end.

metadave commented:
is there a guide on how to submit via Delivery?

metadave commented:
submitting via Delivery :-)

btm committed:
Remove FIPS note that referred to an old release

ChefDK 1.3.40 had a FIPS bug on Windows which was fixed in omnibus-software#823
and released ten days later in ChefDK 1.3.43. This note is cruft and should
have been removed then.

kagarmoe commented:
![image](https://user-images.githubusercontent.com/4400151/27458892-305fe72e-5760-11e7-8190-c5786bbb21b5.png)
